docs/db/migrations.md

```diff
@@ -186,12 +186,14 @@ type User {
 
 #### Default field value
 
+You can specify a default value using the `@default` field annotation as shown below.
```

Comment:
```suggestion
You can specify a default value using the `@default` field marker as shown below.
```

Reply:
We had a discussion yesterday about annotations versus markers and decided to use annotations.

Reply:
ah.. this is confusing as we deprecated `annotations` in the graphql-`metadata`

Reply:
We deprecated `parseAnnotations` - doesnt mean we should retire the use of "annotations" :)